lib/loopback.js

```diff
@@ -76,6 +95,10 @@ function createApplication() {
   app.connector('memory', loopback.Memory);
   app.connector('remote', loopback.Remote);
 
+  // setup the app registry
+  var registry = app.registry = new Registry();
+  require('./builtin-models')(registry);
+
```

Comment:
This should be hidden by a flag, right? And if the flag is not set, then `app.registry = loopback.registry`.

In the future (LB 3.x?) I'd like to stop auto-defining built-in models. This is good for reducing the size of the browser bundle and it will also make it easier to move built-in models to a standalone npm module.

I am proposing to use the opportunity created by implementing local per-app registries and don't include builtin-models in the local registries.